### ISSUE TITLE
Doubts about training data

### ISSUE THREAD

**Issue by xstz** [luannd/MinutiaeNet#12]
I have two question；
1. In the example, the data: - img_files/*.bmp, - mnt_files/*.mnt , - seg_files/*.jpg. is needed when train the corasenet, the orientation field of the fingerprint is not necessary. but the net include a function about enhancement the raw fingerprint image(gaber filter). I don't understand how the gaber filter get the orientation? and there is no loss function for the orientation.
2. The mnt_files/*.mnt file include the orientation about the minutiae. i want to know the format about the 0~ 2*pi ? or others.

**Comment by luannd** [comment]
Sorry for late reply.
1. That function enhances the image to get a robust orientation field. We use that orientation as minutiae orientation in this code.
2. Yes. 0~2pi (if I remember correctly)
